File: src/app/model/prescription.ts
export class Prescription{
    constructor(
        public id:number,
        public name:string,
        public dosage:string,
        public visit_details_id:number,

    ){
        this.id=id ||0;
        this.name=name ||"";
        this.dosage=dosage ||"";
        this.visit_details_id=visit_details_id||0;

    }
}

File: patient-health-record-service/src/main/java/com/revature/patienthealthrecordservice/model/Prescription.java
package com.revature.patienthealthrecordservice.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@Builder
@Table(name = "prescription")
@NoArgsConstructor
@AllArgsConstructor
public class Prescription {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;
	private String name;
	private String dosage;
	private int visit_details_id;

}
